### ISSUE TITLE
Accessibility Insights can not scan app running as system privileges

### ISSUE THREAD

**Issue by 466867517** [robocorp/rpaframework#1138]
Our SAP in windows is Java and need to run as system.

**Comment by mikahanninen** [comment]
Java applications can't be inspected with `Accessibility Insights` (which can be started with admin privileges, but still it won't be able to inspect Java applications).

At the moment Java applications can be inspected with `Assistant` Robot ([link](https://github.com/robocorp/working-with-java)). 

Read more at https://robocorp.com/docs/development-guide/locators/java

**Comment by 466867517** [comment]
> Java applications can't be inspected with `Accessibility Insights` (which can be started with admin privileges, but still it won't be able to inspect Java applications).
> 
> At the moment Java applications can be inspected with `Assistant` Robot ([link](https://github.com/robocorp/working-with-java)).
> 
> Read more at https://robocorp.com/docs/development-guide/locators/java

Yes, I also used this before. But it could not catch the SAP, neither use title nor PID. Even if I use PSTools to start the VScode with system privileges.
Google's Access Bridge Explorer also this situation.
